### DISCUSSION TITLE
FastF1 py-script

### DISCUSSION THREAD

**Discussion by PBrossat** [pyscript/pyscript#1282] [Q&A]
Hello fellow developers,

We are currently facing an issue with the py-script library and we were wondering if someone could lend us a hand. Specifically, we have included the fastF1 library using the .whl file as follows:

<py-config type="json"> 
   { "packages" : ["./static/fastf1-2.3.1-py3-none-any.whl"] } 
</py-config>

The static folder contains the .whl file, but for some reason, it doesn't seem to be working. We were hoping to find out why this might be happening and if anyone else has encountered a similar issue.

We'd like to mention that we were inspired by a project that successfully used the same approach to include another library. However, despite following the same steps, it doesn't seem to be working in our project.

Any help or advice would be greatly appreciated! Thank you in advance.

**Comment by JeffersGlass** [comment]
When you say it's not working - are you presented with any error messages on the page, and/or in the developer console? What exactly are you seeing?

Also, can you share the source of your page here, either in a codeblock or a PasteBin?

**Reply by PBrossat** [reply]
Thanks you for your answer. The problem is now fixed.
